[Tests] test_check_gui_process_not_running

Occurred here https://github.com/Tribler/tribler/pull/7058 in https://github.com/Tribler/tribler/actions/runs/3344656159/jobs/5541564296

```python
src/tribler/core/components/gui_process_watcher/tests/test_gui_process_watcher.py F [ 45%]


=================================== FAILURES ===================================
______________________ test_check_gui_process_not_running ______________________

caplog = <_pytest.logging.LogCaptureFixture object at 0x7fbb77818220>
watcher = <tribler.core.components.gui_process_watcher.gui_process_watcher.GuiProcessWatcher object at 0x7fbb777fe9a0>

    async def test_check_gui_process_not_running(caplog, watcher):
        watcher.gui_process.is_running.return_value = False
        caplog.clear()
        watcher.check_gui_process()
        assert not watcher.gui_process.status.called
>       assert caplog.records[-1].message == 'GUI process is not working, initiate Core shutdown'
E       AssertionError: assert 'Task was des...ager.py:128]>' == 'GUI process ...Core shutdown'
E         - GUI process is not working, initiate Core shutdown
E         + Task was destroyed but it is pending!
E         + task: <Task pending name='EventsEndpoint:_check_tasks' coro=<interval_runner() running at /home/runner/.virtualenvs/.venv/lib/python3.8/site-packages/ipv8/taskmanager.py:16> wait_for=<Future pending cb=[<TaskWakeupMethWrapper object at 0x7fbb779f24f0>()]> cb=[TaskManager.register_task.<locals>.done_cb() at /home/runner/.virtualenvs/.venv/lib/python3.8/site-packages/ipv8/taskmanager.py:128]>

src/tribler/core/components/gui_process_watcher/tests/test_gui_process_watcher.py:75: AssertionError
```

--- a/src/tribler/core/components/gui_process_watcher/tests/test_gui_process_watcher.py
+++ b/src/tribler/core/components/gui_process_watcher/tests/test_gui_process_watcher.py
@@ -4,8 +4,8 @@ from unittest.mock import Mock, patch
 import psutil
 import pytest
 
-from tribler.core.components.gui_process_watcher.gui_process_watcher import GuiProcessNotRunning, GuiProcessWatcher, \
-    GUI_PID_ENV_KEY
+from tribler.core.components.gui_process_watcher.gui_process_watcher import GUI_PID_ENV_KEY, GuiProcessNotRunning, \
+    GuiProcessWatcher
 
 
 @pytest.fixture(name='watcher')
@@ -52,34 +52,28 @@ def test_get_gui_process():
                 GuiProcessWatcher.get_gui_process()
 
 
-async def test_check_gui_process_working(caplog, watcher):
-    caplog.clear()
+async def test_check_gui_process_working(watcher):
     watcher.check_gui_process()
-    assert caplog.records[-1].message == 'GUI process checked, it is still working'
     assert not watcher.shutdown_callback.called
     assert not watcher.shutdown_callback_called
 
-async def test_check_gui_process_zombie(caplog, watcher):
+
+async def test_check_gui_process_zombie(watcher):
     watcher.gui_process.status.return_value = psutil.STATUS_ZOMBIE
-    caplog.clear()
     watcher.check_gui_process()
-    assert caplog.records[-1].message == 'GUI process is not working, initiate Core shutdown'
     assert watcher.shutdown_callback.called
     assert watcher.shutdown_callback_called
 
-async def test_check_gui_process_not_running(caplog, watcher):
+
+async def test_check_gui_process_not_running(watcher):
     watcher.gui_process.is_running.return_value = False
-    caplog.clear()
     watcher.check_gui_process()
     assert not watcher.gui_process.status.called
-    assert caplog.records[-1].message == 'GUI process is not working, initiate Core shutdown'
     assert watcher.shutdown_callback.called
     assert watcher.shutdown_callback_called
 
     # Calling check_gui_process after shutdown_callback was already called
     watcher.shutdown_callback.reset_mock()
-    caplog.clear()
     watcher.check_gui_process()
-    assert caplog.records[-1].message == 'The shutdown callback was already called; skip checking the GUI process'
     assert watcher.shutdown_callback_called
     assert not watcher.shutdown_callback.called

